app/models/UserTeams.js

```diff
@@ -0,0 +1,26 @@
+const db = require("./index");
```

Comment:
Nisam ranije koristio sequelize, ali neka praksa je da se za medju tabele ne prave moduli.
https://sequelize.org/master/manual/advanced-many-to-many.html
U principu bi trebala da mozes iz usera da pristupis listi timova, i obrnuto